apps/fact/fact-frontend/demo.yaml

```diff
@@ -5,6 +5,6 @@ metadata:
 spec:
   values:
     nodejs:
-      image: hmctspublic.azurecr.io/fact/frontend:pr-766-48f8771-20240524104238 #{"$imagepolicy": "flux-system:demo-fact-frontend"}
+      image: hmctspublic.azurecr.io/fact/frontend:pr-784-b9699e2-20240820055029 #{"$imagepolicy": "flux-system:demo-fact-frontend"}
```

Comment:
This will get reverted

You can either, change image policy https://github.com/hmcts/cnp-flux-config/blob/9259532b70b03478cd4a61e3acf55a9e603b19a1/apps/fact/fact-frontend/demo-image-policy.yaml#L7 to be 784. Or remove the image policy from here "flux-system:demo-fact-frontend"}

Change it instead to `"flux-system:fact-frontend"`